devenv/sync.py

```diff
@@ -289,6 +289,7 @@ def main(context: dict[str, str]) -> int:
                 "up",
                 "redis",
                 "postgres",
```

Comment:
should we be bringing up snuba here also for consistency?

Reply:
sure